src/main/webapp/view/transcript/transcript.js

```diff
@@ -112,18 +112,18 @@ function deleteTranscript() {
 /** Seeks transcript to {@code currentTime}, which is given in seconds. */
 function seekTranscript(currentTime) {
   const currentTimeMs = window.secondsToMilliseconds(currentTime);
-  if (currentTimeMs < currentTranscriptLine.startTimestampMs) {
-    return;
+  if (isBeforeTheFirstTranscriptLine(currentTimeMs)) {
+    resetToBeforeTranscript();
   }
   if (isWithinCurrentTimeRange(currentTimeMs)) {
     addBold(currentTranscriptLine);
     return;
   }
+  const nextTranscript = getNextTranscriptLine(currentTimeMs);
   removeBold(currentTranscriptLine);
-  currentTranscriptLine = currentTranscriptLine.nextElementSibling;
```

Comment:
Can you move this into `getNextTranscriptLine` so we don't lose the seeking functionality we already have?  With a TODO to improve/change ofc

Reply:
Done, thanks!